client/a8c-for-agencies/sections/marketplace/hosting-overview-v3/hosting-content/premier-agency-hosting/pressable-plan-section/index.tsx

```diff
@@ -276,8 +298,11 @@ export default function PressablePlanSection( {
 				heading={ translate( 'Schedule a demo' ) }
 				cta={ {
 					label: translate( 'Schedule a demo' ),
-					onClick: () => {},
+					onClick: onScheduleDemo,
+					href: PRESSABLE_CONTACT_LINK,
+					target: '_blank',
```

Comment:
Not part of the main goal but a minor fix that adds a handler for the 'Request demo' button click.